salt/elife-alfred/init.sls

```diff
@@ -455,7 +455,7 @@ siege-log-file:
 
 tox:
     cmd.run:
-        - name: pip install tox==2.9.1
+        - name: python3 -m pip install tox==2.9.1
```

Comment:
could cause fallout on projects that use `tox`, but if they have been ported on Python 3 no problem
e.g. https://github.com/elifesciences/elife-tools, https://github.com/elifesciences/proofreader-python

Reply:
yeah, I know, but I think tox would continue creating Python2.7 environments to test those apps within, right?

Reply:
I think so, as long as `/usr/bin/python2.7` is available. So if, say, `setuptools` needs to do arcane things to compile something for a package it will still do it using in the Python 2.7 virtualenv and shouldn't change behavior.